src/app/MessageDef.h

```diff
@@ -1450,6 +1451,79 @@ class Builder : public ListBuilder
 };
 }; // namespace AttributeDataList
 
+namespace AttributeDataVersionList {
```

Comment:
This looks like a stand-alone class. Lets move it to a standalone header (and likely .cpp file as well)

Reply:
yep

Reply:
per offline chat, will do the refactor #3644 in this pr

Reply:
Done, thanks